### ISSUE TITLE
Selecting pure import and jQuery Mobile can cause an internal copy error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#54]
```
If the source already has a jquery.mobile directory and jQuery Mobile is 
selected along with "Pure Import", an internal copy error will be generated.

I'm not sure it's a valid use case to select jQuery Mobile and Pure Import, but 
it still shouldn't generate the error.

If a user wants to add in jQuery Mobile along with the one already in the 
source directory, s/he should rename it first.


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 21 Dec 2011 at 2:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 21 Dec 2011 at 2:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 3 Feb 2012 at 9:39
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 3 Feb 2012 at 9:39
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
